packages/wasm/package.json

```diff
@@ -7,19 +7,19 @@
     "test": "vitest run"
   },
   "dependencies": {
-    "@bufbuild/protobuf": "^1.5.1",
+    "@bufbuild/protobuf": "^1.6.0",
     "@penumbra-zone/types": "workspace:*",
-    "@penumbra-zone/wasm-bundler": "0.64.0",
-    "@penumbra-zone/wasm-nodejs": "0.64.0",
+    "@penumbra-zone/wasm-bundler": "file:../../../penumbra/crates/wasm/publish/bundler/penumbra-zone-wasm-bundler-0.64.1.tgz",
```

Comment:
change to `0.64.1-preview-3`

Reply:
does it certainly work? there's new commits since then

Reply:
actually it doesn't. I can run `pnpm install` with `file:../../../penumbra/crates/wasm/publish/bundler/penumbra-zone-wasm-bundler-0.64.1.tgz`

Reply:
It doesn't, we'll need to publish `0.64.1-preview-4` from main.